modules/core/include/opencv2/core/mat.hpp

```diff
@@ -1970,9 +1970,11 @@ class CV_EXPORTS Mat
     //! pointer to the data
     uchar* data;
 
-    //! helper fields used in locateROI and adjustROI
+    //! beginning of (parent) matrix -> needed when submatrix
```

Comment:
I believe, the language should be improved here, now it does not look like proper English